src/main/scala/nl.knaw.dans.easy.dd2d/Deposit.scala

```diff
@@ -71,11 +71,18 @@ case class Deposit(dir: File) extends DebugEnhancedLogging {
     depositProperties.getString("identifier.doi", "")
   }
 
+  def isUpdate: Try[Boolean] = {
+    for {
+      bag <- tryBag
+      isVersionOf = bag.getMetadata.get("Is-Version-Of")
+    } yield isVersionOf != null && isVersionOf.size() > 0
```

Comment:
perhaps: `Option(isVersionOf).toSeq.flatten.nonEmpty`

Reply:
Is that really more readable?

Reply:
That is the reason for "perhaps"